test/client/index.js

```diff
@@ -272,6 +273,46 @@ Client.prototype.keys = function () {
     )
 }
 
+Client.prototype.devices = function () {
+  var o = this.sessionToken ? P.resolve(null) : this.login()
+  return o.then(
+    function () {
+      return this.api.accountDevices(this.sessionToken)
+    }.bind(this)
+  )
+}
+
+Client.prototype.updateDevice = function (info) {
+  var o = this.sessionToken ? P.resolve(null) : this.login()
+  return o.then(
+    function () {
+      return this.api.updateDevice(this.sessionToken, info)
+    }.bind(this)
+  )
+  .then(
+    function (device) {
+      if (this.device.id === device.id) {
+        this.device = device
+      }
+      return device
+    }.bind(this)
+  )
+}
+
+Client.prototype.deleteDevice = function (id) {
```

Comment:
By contrast, the names in this file are "verbNoun" like "destroyAccount".  I've no idea why...